components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -2509,6 +2508,203 @@ public void updateSubscriptionStatusAndTier(int subscriptionId, String status) t
         }
     }
 
+    /**
+     * This method is used to obtain subscribers for a given organization
+     *
+     * @param organization Organization
+     * @throws APIManagementException if failed to obtain subscribers
+     */
```

Comment:
Shall we please have a seperate DAO inside the cleanup package and move all these logic?
ApiMgtDAO is already gone beyond 17000 LoC :disappointed: